sdk/core/azure-core-tracing-opentelemetry/test/ut/azure_core_test.cpp

```diff
@@ -10,6 +14,17 @@ int main(int argc, char** argv)
   // End users need to decide if SIGPIPE should be ignored or not.
   signal(SIGPIPE, SIG_IGN);
 #endif
+  // Declare a signal handler to report unhandled exceptions on Windows - this is not needed for
+  // other
+// OS's as they will print the exception to stderr in their terminate() function.
```

Comment:
The suggestion to indent the comment still applies. But also, the line above just has "// other" on it, so there is an explicit line break that needs to be removed.

```suggestion
  // other OS's as they will print the exception to stderr in their terminate() function.
```